CMakeLists.txt

```diff
@@ -463,9 +459,9 @@ add_library(glog::glog ALIAS glog)
 
 set_target_properties (glog PROPERTIES POSITION_INDEPENDENT_CODE ON)
 
-if (UNWIND_LIBRARY)
-  target_link_libraries (glog PUBLIC ${UNWIND_LIBRARY})
-endif (UNWIND_LIBRARY)
+if (Libunwind_FOUND)
+  target_link_libraries (glog PUBLIC unwind)
```

Comment:
Libunwind should export an import target in `package::lib` format. I suggest you use `unwind::unwind`.